test/MLIR/NativeSpec.hs

```diff
@@ -85,16 +90,16 @@ spec = do
         m <- MLIR.createEmptyModule loc
         str <- (MLIR.moduleAsOperation >=> MLIR.showOperationWithLocation) m
         MLIR.destroyModule m
-        str `shouldBe` "module  {\n} loc(#loc)\n#loc = loc(\"test.cc\":21:45)\n"
+        str `bsShouldContain` "loc(\"test.cc\":21:45)"
 
     it "Can create an empty module with name location" $ \ctx -> do
       MLIR.withStringRef "WhatIamCalled" $ \nameRef -> do
-        childLoc <- MLIR.getUnknownLocation ctx
-        loc <- MLIR.getNameLocation ctx nameRef childLoc
+        -- childLoc <- MLIR.getUnknownLocation ctx
+        loc <- MLIR.getNameLocation ctx nameRef =<< MLIR.getUnknownLocation ctx
         m <- MLIR.createEmptyModule loc
         str <- (MLIR.moduleAsOperation >=> MLIR.showOperationWithLocation) m
         MLIR.destroyModule m
-        str `shouldBe` "module  {\n} loc(#loc)\n#loc = loc(\"WhatIamCalled\")\n"
+        str `bsShouldContain` "loc(\"WhatIamCalled\")"
```

Comment:
I think it's ok if you just call this `shouldContain` without the `bs` prefix. If we ever need to make it polymorphic we can always turn it into a type-class or add the prefixes then.